packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -186,9 +186,12 @@ export const ApplicationMenu = (props: ApplicationMenuProps) => {
 
       const [entry] = entries;
 
+      /** Adding a 12 pixel (spacing-25) buffer to the height calculation when determining if the submenu fits within the viewport below the menu item,
+       * this value accounts for the padding introduced on link hover. We need this to account for the additional card height when a hovered submenu text link wraps to the next line*/
       const doesSubmenuFitWithinViewportBelowMenuItem =
         entry.boundingClientRect.height +
-          (props.isMenuOpen ? menuItemTop : menuItemBottom) >
+          (props.isMenuOpen ? menuItemTop : menuItemBottom) +
+          12 >
```

Comment:
Maybe we could use here `const SINGLE_LINE_HEIGHT_BUFFER = 12;`

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3600/commits/a5e97c3d1c08fd45cc8e8e31c76f2981029b129f

Thank you